src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -17,13 +17,15 @@
 public class DriverDaoImpl implements DriverDao {
```

Comment:
Let's also move your implementations to the impl folder

Reply:
Just left it as original... but ok again)